README.md

```diff
@@ -6,10 +6,83 @@
 
 ## Introduction
 
-Frequenz gRPC API for monitoring and control of microgrids.
+The Frequenz Microgrid API is an API for operating and monitoring electrical
+components and sensors within a microgrid. It offers various functionalities
+related to microgrid management, including metadata retrieval, component
+listing, setting power levels, and controlling the state of components. The API
```

Comment:
The service .... instead of The API I'd suggest there too.

Reply:
I'd try to use the term "API" more than "service" in the docs, but if you have a strong feeling about this, I can change it to "service".

Reply:
I was about to say the same, but I let it pass because I also don't find it super important, but from "service" I understand the daemon implementing the API. To avoid the redundancy in the text we could just say: "The Frequenz Microgrid API defines the interactions to operate and monitor electrical
components and sensors within a microgrid. It offers various functionalities
related to microgrid management, including metadata retrieval, ..." (or something like that)